### PR TITLE
feat: implement PastFestival, ThisYear, Footer, FAQs and AboutCMFest …

### PR DESCRIPTION
**Pull request by egokiemute**
…components

### CODE REVIEW

File: astro.config.mjs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { defineConfig } from 'astro/config';
```
Comment (codejagaban) [inline astro.config.mjs]: Why are we using this exactly?

File: public/chevron-down.svg
Comment (codejagaban) [inline public/chevron-down.svg:1]: we should change all of these icons to real sponsors

File: src/components/AccordionItem.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { useState } from "react";
+
+const AccordionItem = ({ question, answer }: { question: string; answer: string }) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleClick = () => {
+    console.log("Accordion clicked");
```
Comment (codejagaban) [inline src/components/AccordionItem.tsx]: remove logs

File: src/components/AccordionItem.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    setIsOpen(prevState => !prevState);
+  };
+
+  return (
+    <div className="py-6 border-b border-bg-primary/30 cursor-pointer" onClick={handleClick}>
+      <div className="flex items-center justify-between">
+        <span className="flex-1 text-lg font-bold pr-3 block max-w-sm md:w-full">{question}</span>
```
Comment (codejagaban) [inline src/components/AccordionItem.tsx]: The length of the question is too short and doesn't fit design.

Screenshot 2024-08-11 at 1 32 39 PM

File: src/components/Speaker.tsx
Comment (codejagaban) [inline src/components/Speaker.tsx:1]: speaker component not working

File: src/layouts/Header.tsx [hunk 4]
```diff
@@ -52,6 +52,9 @@ export default function Header() {
                 <a href="#" onClick={() => setToggle(false)}>
                   Agenda
                 </a>
+                <a href="#" onClick={() => setToggle(false)}>
```
Comment (codejagaban) [inline src/layouts/Header.tsx]: Link it to FAQ section

File: src/sections/ThisYear.astro [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    </div>
+    <Button
+      body="Buy your Tickets"
+      icon="/arrow_right_up.svg"
+      className="bg-goldenrod text-text-secondary font-semibold px-4 py-2 border-none rounded-md"
+    />
+  </div>
+  <div class="flex justify-center items-center w-full h-[520px] relative">
+    <div
+      class="flex justify-center items-end w-full h-96 bg-this-year rounded-xl mt-10"
```
Comment (codejagaban) [inline src/sections/ThisYear.astro]: why is this image a background image? Also the width of the image and the text doesn't fit the design

**Review by codejagaban — CHANGES_REQUESTED** [review]

File: astro.config.mjs (hunk 1 above)
Comment (egokiemute) [reply astro.config.mjs]: I updated the astro config file resolve this particular issue

"Could not process image request: MissingSharp: Could not find Sharp. Please install Sharp (`sharp`) manually into your project or migrate to another image service."

File: src/components/AccordionItem.tsx (hunk 2 above)
Comment (egokiemute) [reply src/components/AccordionItem.tsx]: Removed

File: src/components/AccordionItem.tsx (hunk 3 above)
Comment (egokiemute) [reply src/components/AccordionItem.tsx]: Fixed

File: src/components/Speaker.tsx
Comment (egokiemute) [reply src/components/Speaker.tsx:1]: I commented it out still working on it

File: src/layouts/Header.tsx (hunk 4 above)
Comment (egokiemute) [reply src/layouts/Header.tsx]: Done

File: public/chevron-down.svg
Comment (egokiemute) [reply public/chevron-down.svg:1]: I believe you meant the Sponsors

But the Figma file has not been updated with the real images and content also.

File: astro.config.mjs (hunk 1 above)
Comment (codejagaban) [reply astro.config.mjs]: I'm not getting this error from my end.

What version of Node.js are you using?

File: public/chevron-down.svg
Comment (codejagaban) [reply public/chevron-down.svg:1]: Will share the content with you then

File: astro.config.mjs (hunk 1 above)
Comment (egokiemute) [reply astro.config.mjs]: 18.19.0

File: public/chevron-down.svg
Comment (egokiemute) [reply public/chevron-down.svg:1]: Okay

**Review by codejagaban — APPROVED** [review]
Looks good.

Thanks

**Pull request merged**